### ISSUE TITLE
Why not divide the training set and test set in the 3d-gan?

### ISSUE THREAD

**Issue by Yuzikun-gu** [jaikanthjay46/HSI-Classification-GAN#2]

**Comment by jaikanthjay46** [comment]
3D GAN had train test split, It got missed somehow, please see commits [bc46e](https://github.com/jaikanthjay46/HSI-Classification-GAN/commit/bc46e9c2234cc7ab346effbdd3883da6b065e298) and [c62af](https://github.com/jaikanthjay46/HSI-Classification-GAN/commit/4a2b83e380393c87c62afab67629273342a36021)
